src/middlewares/roomsAvailability.js

```diff
@@ -24,8 +49,8 @@ const checkForRooms = async (req, res, next) => {
   });
 
   if (unavailableRooms.count > 0) {
-    const bookedRooms = unavailableRooms.rows.map(room => room.id);
-    return Responses.handleSuccess('409', 'rooms already booked', res, { unAvailableRooms: bookedRooms });
+    const bookedRoom = unavailableRooms.rows.map(room => room.id);
+    return Responses.handleSuccess('409', 'rooms already booked', res, { unAvailableRooms: bookedRoom });
```

Comment:
The response code has to be a number `'409'` to `409`

Reply:
> Great work Shaphan. I tested and it works fine however a room can be booked even when it doesn't exist in the database. Maybe you can add a middleware to check if the room exists first then we'll be good to go.

Thanks @Mastel22 for finding out this, now solved.